Server/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.cpp

```diff
@@ -1431,6 +1477,31 @@ int CLuaPlayerDefs::ResendPlayerACInfo(lua_State* luaVM)
     return 1;
 }
 
+int CLuaPlayerDefs::GetPlayerScriptDebugLevel(lua_State* luaVM)
+{
+    // int getPlayerScriptDebugLevel ( player thePlayer )
+    CPlayer* pPlayer;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pPlayer);
+
+    if (!argStream.HasErrors())
```

Comment:
As per #821 we should do a hard error here like so:
```cpp
if (argStream.HasErrors())
{
    return luaL_error(luaVM, argStream.GetFullErrorMessage());
}
```
and then move the passing code after that.